kubernetes/aws-eks-deployment.md

```diff
@@ -0,0 +1,167 @@
+# Deploying Gaffer on AWS EKS
+
+The following instructions will guide you through provisioning and configuring an [AWS EKS](https://aws.amazon.com/eks/) cluster that our Helm Charts can be deployed on.
+
+
+## Install CLI Tools
+
+* [docker-compose](https://github.com/docker/compose/releases/latest)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [helm](https://github.com/helm/helm/releases)
+* [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [eksctl](https://github.com/weaveworks/eksctl/releases/latest)
+
+
+## Container Images
+
+If the versions of the container images you would like to deploy are not available in [Docker Hub](https://hub.docker.com/u/gchq) then you will need to host them in a registry yourself.
+
+The following instructions build all the container images and host them in AWS ECR:
+
+```
```

Comment:
Might be a good idea to use the bash syntax highlighting here and in other places to help reading. 
````suggestion
bash
```
````